docs/compilers/CSharp/CodeGen Differences.md

```diff
@@ -17,58 +16,59 @@ There are two sections here - Release codegen and Debug. They are separate becau
  
-�	Lambdas had some minor changes in caching strategy and a change in representation of non-lifting lambdas.
-- The caching strategy has changed to a more compact pattern that also results in fewer accesses to the caching field (in case JIT does not optimize multiple reads). 
-Instead of an equivalent of 
+*	Lambdas had some minor changes in caching strategy and a change in representation of non-lifting lambdas.
+    - The caching strategy has changed to a more compact pattern that also results in fewer accesses to the caching field (in case JIT does not optimize multiple reads). 
+Instead of an equivalent of:
  
-```C#
- if (cacheField == null)
- {
-   cacheFiled = {allocate new lambdaDelegate};
- }
- return cacheField;
-```
-We now generate something close to
+        ```C#
+        if (cacheField == null)
+        {
+          cacheFiled = {allocate new lambdaDelegate};
+        }
+        return cacheField;
+        ```
+
+        We now generate something close to:
 
-```C#
-return cacheField ?? (cacheFiled = {allocate new lambdaDelegate});
-```
+        ```C#
+        return cacheField ?? (cacheFiled = {allocate new lambdaDelegate});
+        ```
 
-- non-lifting Lambda expressions are now implemented as an instance methods on singleton display classes. Since the entry point to the delegate is the instance "Invoke" method, it is cheaper at runtime to dispatch delegate invocations to the underlying implementing method if such method is also an instance method with exactly same formal signature as "Invoke".
-- delegates for non-lifting Lambdas in generic methods can now be cached by Roslyn. That is mostly a welcome sideeffect of the change above.
+    - non-lifting Lambda expressions are now implemented as an instance methods on singleton display classes. Since the entry point to the delegate is the instance "Invoke" method, it is cheaper at runtime to dispatch delegate invocations to the underlying implementing method if such method is also an instance method with exactly same formal signature as "Invoke".
+    - delegates for non-lifting Lambdas in generic methods can now be cached by Roslyn. That is mostly a welcome sideeffect of the change above.
 
-�	The string switch codegen in Rolsyn is completely new. Roslyn does not use dictionaries to avoid allocations and a potentially huge penalty when a string switch is execute for the first time. Roslyn uses a private function that maps strings to hash codes and a numeric switch. In some sense this is a partial inlining of the former technic that used static dictionaries.
+*	The string switch codegen in Rolsyn is completely new. Roslyn does not use dictionaries to avoid allocations and a potentially huge penalty when a string switch is executed for the first time. Roslyn uses a private function that maps strings to hash codes and a numeric switch. In some sense this is a partial inlining of the former technic that used static dictionaries.
```

Comment:
```suggestion
* The string switch codegen in Rolsyn is completely new. Roslyn does not use dictionaries to avoid allocations and a potentially huge penalty when a string switch is executed for the first time. Roslyn uses a private function that maps strings to hash codes and a numeric switch. In some sense this is a partial inlining of the former technic that used static dictionaries.
```